fdmi/source_dock_fom.c

```diff
@@ -630,22 +633,46 @@ static int node_eval(void                        *data,
 	return src_rec->fsr_src->fs_node_eval(src_rec, value_desc, value);
 }
 
+static struct m0_fdmi_sd_filter_type_handler fdmi_filter_type_handlers[] = {
+	{
+		.ffth_id      = M0_FDMI_FILTER_TYPE_TREE,
+		.ffth_handler = &m0_fdmi_eval_flt,
```

Comment:
Can we rename the handler to:
```
m0_fol_fdmi_filter_tree
```
So the name matches the type id and is similar to m0_fol_fdmi_filter_kv_substring()

Reply:
I just tried to do this and it seems like we have the following:

- `fdmi/flt_eval.h`, which has `m0_fdmi_eval_*()` functions. If I rename it to `m0_fol_fdmi_filter_kv_substring()` then the naming of the functions there would be non-uniform and not according to our coding style;
- `m0_fdmi_eval_flt()` is not specific to `fol`, so adding `fol` to the name would only add confision.

So I decided to not to rename. Please let me know if this is OK.

Reply:
OK